tools/clang/test/hlsl_cmds_test/lit.cfg

```diff
@@ -0,0 +1,91 @@
+
+import os
+import platform
+import re
+import subprocess
+import tempfile
+
+import lit.formats
+import lit.util
+
+config.name = 'hlsl-cmds'
+
+
+# Check that the object root is known.
+llvm_obj_root = getattr(config, 'llvm_obj_root', None)
+if llvm_obj_root is None:
+    # Otherwise, we haven't loaded the site specific configuration (the user is
+    # probably trying to run on a test file directly, and either the site
+    # configuration hasn't been created by the build system, or we are in an
+    # out-of-tree build situation).
+
+    # Check for 'clang_unit_site_config' user parameter, and use that if available.
+    site_cfg = lit_config.params.get('clang_hlsl_cmds_site_config', None)
+    if site_cfg and os.path.exists(site_cfg):
+        lit_config.load_config(config, site_cfg)
+        raise SystemExit
+
+if config.spirv:
+    config.available_features.add("spirv")
+else:
+    config.available_features.add("nospirv")
+
+
+bin_dir = os.path.join(config.llvm_obj_root, lit_config.params['build_mode'], 'bin')
+dxc_bin = os.path.join(bin_dir, 'dxc.exe')
+
+config.environment['PATH'] = str.format('{};{}',config.environment['PATH'], bin_dir)
+
+config.substitutions.append( ('%dxc',
```

Comment:
It makes sense to add these substitutions to the top-level lit config (dxc is already there).